src/index.ts

```diff
@@ -1600,6 +1639,16 @@ type FilterFunctionKey =
 	| '$cast';
 type FilterFunctionValue = Filter;
 
+type DurationValue =
+	| string
+	| {
+			negative?: boolean;
+			day?: number;
+			hour?: number;
+			minute?: number;
+			second?: number;
```

Comment:
@Page-  The abstract sql compiler has these properties in signular, so I kept it consistent.
How did we originally end up using singular names there? Was it for consistency with the `($)minute,second,...` OData function?

I really think though that the plural version seems better, so let me know if you agree.
```suggestion
			days?: number;
			hours?: number;
			minutes?: number;
			seconds?: number;
```

Reply:
I'm happy with the plural versions